user_guide/integrations_overview.rst

```diff
@@ -5,8 +5,7 @@ OroCRM Integrations Overview
 
 OroCRM supports two types of integrations: 
 
-- Integrations applied to the whole system. There can only be one integration with a third-party system for an OroCRM 
-  instance. These integrations can be enabled/disabled and defined in the *"System → Configuration → Integration"*.
+- Integrations are applied system wide. An instance of an integration, such as with a third-party system, can only be applied once per OroCRM instance. These integrations can be enabled/disabled and defined in the *"System → Configuration → Integration"*.
```

Comment:
@sergeyz are you suggesting to remove `such as with a third-party system` ?

How can a user create multiple connections to a 3rd part system using the same integration?  (Understanding that you duplicate the bundles... but based previous explanations, an Integration can only be "activated" once per orocrm instance.  This correct?

I use the correct out of the box / demo UI as the example:
![screen shot 2016-02-01 at 2 08 18 pm](https://cloud.githubusercontent.com/assets/1994838/12728017/cb3fa3fe-c8ed-11e5-8ba1-17326b40f802.png)

The user cannot add multiple google single sign on credentials.